app/src/auto_validator/core/api.py

```diff
@@ -62,20 +68,15 @@ class DumperCommandsViewSet(viewsets.ViewSet):
     def list(self, request):
         subnet_identifier = request.headers.get("SubnetID")
         if not subnet_identifier:
-            return Response({"error": "subnet_identifier is required"}, status=status.HTTP_400_BAD_REQUEST)
-
-        try:
-            with open(YAML_FILE_PATH) as file:
-                data = yaml.safe_load(file)
-                codename_lower = subnet_identifier.lower()
-                for normalized_codename, sn_config in data.items():
-                    codenames = sn_config.get("codename_list", [])
-                    if codename_lower in map(str.lower, codenames):
-                        return Response((normalized_codename, sn_config.get("dumper_commands", [])))
-                else:
-                    return Response({"error": "subnet_identifier not found"}, status=status.HTTP_404_NOT_FOUND)
-        except Exception as e:
-            return Response({"error": str(e)}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
+            return Response({"error": "SubnetID is required"}, status=status.HTTP_400_BAD_REQUEST)
+
+        dumper_commands = get_dumper_commands(subnet_identifier, SUBNETS_CONFIG_PATH)
+        if dumper_commands is not None:
+            logger.info(f"SubnetID: {subnet_identifier}, dumper_commands: {dumper_commands}")
```

Comment:
```suggestion
            logger.info("SubnetID: %s, dumper_commands: %s", subnet_identifier, dumper_commands)
```
this is the only way in which you should be formatting log statements. Long story, just trust me or ask some other day.